README.md

```diff
@@ -1,10 +1,10 @@
 # TM4J Jenkins Plugin
 
-A plugin for Jenkins built for integrating automated tests with [Test Management for Jira Server](https://marketplace.atlassian.com/apps/1213259/test-management-for-jira?hosting=server&tab=overview) (TM4J).  
+A plugin for Jenkins built for integrating automated tests with [Test Management for Jira Server](https://marketplace.atlassian.com/apps/1213259/test-management-for-jira?hosting=server&tab=overview) and [Test Management for Jira Cloud](https://marketplace.atlassian.com/apps/1213259/tm4j-test-management-for-jira?tab=overview&hosting=cloud) (TM4J).  
```

Comment:
The last "(TM4J)" is a bit lost there. Suggestion:

A plugin for Jenkins built for integrating automated tests with Test Management for Jira (TM4J). Compatible with both Jira Server and Jira Cloud. For more information, please check [TM4J Server](https://marketplace.atlassian.com/apps/1213259/test-management-for-jira?hosting=server&tab=overview) and [TM4J Cloud](https://marketplace.atlassian.com/apps/1213259/tm4j-test-management-for-jira?tab=overview&hosting=cloud).